src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,27 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(4, $full-card-width);
+  gap: 48px;
+  justify-content: center;
+
+  width: ($full-card-width * 4) + ($card-gap * 3);
+
+  margin: 0 auto;
+  padding-top: 50px;
+
+  @media (max-width: 1024px) {
```

Comment:
try to realize it by min-width, this way you can drop grid-template-columns: repeat(4, $full-card-width) from .container and just set in media